apps/council-ui/src/ui/push/subscribePush.ts

```diff
@@ -0,0 +1,115 @@
+import * as PushAPI from "@pushprotocol/restapi";
+import { SubscribeOptionsType } from "@pushprotocol/restapi/src/lib/channels";
+import { useCallback, useEffect, useState } from "react";
+
+import { useChainId } from "src/ui/network/useChainId";
+import { useAccount, useSignTypedData } from "wagmi";
+import PushSettings from "./settings.json";
+import { UsePushSubscribeType } from "./types";
+
+export function usePushSubscribe(): UsePushSubscribeType {
+  const { address } = useAccount();
+  const chainId = useChainId();
+  const [loading, setLoading] = useState(false);
+  const [isSubscribed, setIsSubscribed] = useState(false);
+
+  const fetchChainId = () => {
```

Comment:
Since we're using function declarations everywhere else (i.e., `function fetchChainId()`) let's change this one to be consistent. Same with all other instances of `const ... = () =>`